tests/plugins/tts/FestivalTTSEngine.test.js

```diff
@@ -0,0 +1,70 @@
+import '../../../BookReader/jquery-1.10.1.js';
+import '../../../BookReader/jquery.browser.min.js';
+import FestivalTTSEngine from '../../../src/js/plugins/tts/FestivalTTSEngine.js';
+import sinon from 'sinon';
+import { afterEventLoop } from '../../utils.js';
+/** @typedef {import('../../../src/js/plugins/tts/AbstractTTSEngine.js').TTSEngineOptions} TTSEngineOptions */
+
+describe('iOSCaptureUserIntentHack', () => {
+    test('synchronously calls createSound/play to capture user intent', () => {
+        const mockSound = createMockSound();
+        const sm = {
+            supported: sinon.stub().returns(true),
+            createSound: sinon.stub().returns(mockSound)
```

Comment:
SUGGESTION: For AirBNB style, add a comma.

Reply:
Done